packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethods.css

```diff
@@ -13,5 +13,5 @@
 
 .radio_label {
     font-weight: 600;
-    width: 6rem;
+    min-width: 16rem;
```

Comment:
Stretched to fit longer text since it comes from the back end. This doesn't account for wrapping text though. Any ideas on how to handle this more gracefully would be welcome :)

*Before*
[![Image from Gyazo](https://i.gyazo.com/f0c7092b7ffe2c0d2b8bcd9b874485fd.png)](https://gyazo.com/f0c7092b7ffe2c0d2b8bcd9b874485fd)

*After*
[![Image from Gyazo](https://i.gyazo.com/d3afd612df1d6c9a863707df61c5c9e2.png)](https://gyazo.com/d3afd612df1d6c9a863707df61c5c9e2)

Reply:
Looks better. It just needs a bit more `line-height` in case it does wrap.